devel/webify/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/beefsack/webify 1.5.0 v
+categories          devel
+license             MIT
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+description         Turn shell commands into web services
+long_description    ${description}
+
+checksums           ${distname}${extract.suffix} \
```

Comment:
Hey @harens - thanks for this submission.

Though I've often made the same mistake, if you have a single distfile, then you don't actually need to include the `${distname}${extract.suffix}` label in the `checksums` block.  Just the checksums should be fine.

You'd add that in addition to other filenames if your port needs to download & checksum multiple distfiles in order to perform the build.

Reply:
Thanks as always for your help @herbygillot. After https://github.com/macports/macports-ports/pull/8337#issuecomment-689716937, I have not added a commit msg.